api/instance/v1/security_group_utils_test.go

```diff
@@ -137,7 +137,7 @@ func TestAPI_UpdateSecurityGroupRule(t *testing.T) {
 
 		testhelpers.AssertNoError(t, err)
 		testhelpers.Equals(t, SecurityGroupRuleActionDrop, updateResponse.Rule.Action)
-		testhelpers.Equals(t, scw.IPNet{IPNet: net.IPNet{IP: net.IP{0x1, 0x1, 0x1, 0x1}, Mask: net.IPMask{0xff, 0xff, 0xff, 0xff}}}, updateResponse.Rule.IPRange)
+		testhelpers.Equals(t, scw.IPNet{IPNet: net.IPNet{IP: net.IP{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 0x1, 0x1, 0x1, 0x1}, Mask: net.IPMask{0xff, 0xff, 0xff, 0xff}}}, updateResponse.Rule.IPRange)
```

Comment:
Why do you need an IPv6 here?

Reply:
I just add the missing bytes that are returned now

Reply:
These bytes are corresponding to an IPv6, not an IPv4

Reply:
I've juste apply net.IPv4 to be more explanatory.